syntax = "proto3";
  
package caber.macaron;

import "google/protobuf/any.proto";

// Generic parsers to parse the input data source.
message DataSourceConfig {
  // Name of the parser class.
	string class_name = 1;

	// Options for the parser class.
	google.protobuf.Any class_options = 2;
}
